scripts/sandbox_start.sh

```diff
@@ -8,20 +8,77 @@ set -e
 
 echo "Running $0"
 
-
-
 setupFabric() {
 # wait until peer is up
 # TODO we need health check for peer
 sleep 20
 cd test/bdd && go test
 }
 
+registerRPTenant() {
```

Comment:
we don't add setup scripts part of sandbox_start.sh because the ops will not use the sandbox_start.sh

Reply:
please take look at this https://github.com/trustbloc/edge-sandbox/blob/master/test/bdd/fixtures/scripts/vcs_verifier_configure.sh#L12

Reply:
@fqutishat 

> we don't add setup scripts part of sandbox_start.sh because the ops will not use the sandbox_start.sh

What's the alternative? `rp.adapter.example.com` needs to be up in order to register `rp.example.com` as a tenant, then the latter needs to be configured with the generated oauth2 parameters.

Reply:
@llorllale we have same issue for edge-service please take look at this solution https://github.com/trustbloc/edge-sandbox/blob/master/test/bdd/fixtures/demo/docker-compose-edge-components.yml#L35

Reply:
@fqutishat i'll make it part of the startup of  `rp.example.com`

Reply:
@fqutishat please check now